buildenv/jenkins/JenkinsfileBase

```diff
@@ -656,11 +663,18 @@ def buildTest() {
 			// check for each lib. If lib does not exist, donwload it.
 			// If lib exists, SHA will be checked. Re-download if SHA does not match.
 			timeout(time: 20, unit: 'MINUTES') {
-				def customUrl = "https://ci.adoptium.net/job/test.getDependency/lastSuccessfulBuild/artifact/"
+				def defaultUrl = "https://ci.adoptium.net/job/test.getDependency/lastSuccessfulBuild/artifact/"
+				def systemUrl = "https://ci.adoptium.net/job/systemtest.getDependency/lastSuccessfulBuild/artifact/"
+				env.LIB_DIR = (env.BUILD_LIST == 'system') ? env.SYSTEM_LIB_DIR : env.LIB_DIR
+				def customUrl = (env.BUILD_LIST == 'system') ? systemUrl : defaultUrl
```

Comment:
I would prefer to have if statement to make the logic clear:

```
if (env.BUILD_LIST == 'system') {
   env.LIB_DIR = env.SYSTEM_LIB_DIR
   customUrl = "https://ci.adoptium.net/job/systemtest.getDependency/lastSuccessfulBuild/artifact/"
} else {
   customUrl = "https://ci.adoptium.net/job/test.getDependency/lastSuccessfulBuild/artifact/"
}

```

Please note this change will conflict with https://github.com/adoptium/aqa-tests/pull/5289. Depends on which PR gets merged first, the other one will need to be updated.